velox/vector/tests/utils/VectorTestBase.h

```diff
@@ -459,6 +459,21 @@ class VectorTestBase {
         size, sizeAt, valueAt, isNullAt, valueIsNullAt, arrayType);
   }
 
+  /// Similar to makeArrayVectorFromJson, Creates an ArrayVector from list of
+  /// JSON arrays of arrays.
+  /// @tparam T Type of array elements. Must be an integer: int8_t, int16_t,
+  /// int32_t, int64_t.
+  /// @param jsonArrays A list of JSON arrays. JSON array cannot be an empty
+  /// string.
+  /// @param arrayType type of array elements. Defaults to Array of T type
```

Comment:
> Defaults to Array of T type if not specified.

This statement is not accurate and redundant as it simply repeats the code. Let's remove. Array of T is not the same as ARRAY(CppToType<T>::create()).